app/javascript/styles/fonts/montserrat.scss

```diff
@@ -0,0 +1,11 @@
+@font-face {
```

Comment:
Same question here ... it seems weird that the fonts move into `app/javascript/...` -- is that normal?